python/mxnet/ndarray/ndarray.py

```diff
@@ -5087,7 +5087,7 @@ def _make_dl_managed_tensor(array):
         return array(ndarray, dtype=ndarray.dtype)
 
     if not ndarray.flags['C_CONTIGUOUS']:
-        raise ValueError("Only c-contiguous arrays are supported for zero-copy")
+        return array(ndarray, dtype=ndarray.dtype)
```

Comment:
Why does this PR affect this line?

Reply:
The same thing could actually be done in another way, I can catch this error outside the function and handle accordingly.